pySDC/core/BaseTransfer.py

```diff
@@ -45,7 +45,6 @@ def __init__(self, fine_level, coarse_level, base_transfer_params, space_transfe
         # set up logger
         self.logger = logging.getLogger('transfer')
 
-        # just copy by object
```

Comment:
It's not a copy, but rather the assignment of `self.fine` to the variable referenced by `fine_level`. Python never use copy except if you tell him that explicitly, or create new objects before assigning